gatling/user-files/DockerFile/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM denvazh/gatling
+
+ENV NODE_VERSION=10.15.1
+ENV PATH=/opt/node-v$NODE_VERSION-linux-x64/bin:$PATH
+
+ADD https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION.tar.gz /tmp/
+
+RUN apk add build-base python python-dev py-pip linux-headers nodejs-npm\
+    && npm install --save request \
```

Comment:
Why do we install `request` and `request-promise` here and, more importantly, where are they installed?

Either you need them globally and this is the right place to install those, but you need the `-g` option, or you don't and a `package.json` in the benchmark repo is sufficient.
My gut tells me that the second option is preferable.